code/modules/mob/living/living_update_icons.dm

```diff
@@ -0,0 +1,17 @@
+
+/// Called whenever the mob is to be resized or when lying/standing up for carbons.
+/mob/living/update_transform()
```

Comment:
name this proc something like `handle_transform_update` since it's handling the update of the transform.

Reply:
Would it be ok if I renamed the other one like this?

Reply:
the other one isn't updating the transform though, this is handling the transform update so it would be misleading

Reply:
so renaming all update_transform calls it's.

Reply:
any name is fine, just don't have it as _update_transform and avoid making the names misleading.  You can keep this as update_transform if you can find an appropriate name for the other proc.